enterprise-suite/values.yaml

```diff
@@ -17,6 +17,38 @@ configMapReloadVersion: v0.2.2
 kubeStateMetricsVersion: v1.2.0
 # prom/node-exporter
 nodeExporterVersion: v0.15.2
+# janeczku/go-dnsmasq
+goDnsmasqVersion: release-1.0.7
+# alpine
+alpineVersion: latest
+# busybox
+busyboxVersion: latest
+
+#################################################
+# Image names to use. Combined with the versions above, gives the fully qualified docker image name.
```

Comment:
Do we want to list the names before the images?

Reply:
I'm not sure I follow. Can you give me an example?

Reply:
We list versions starting at line 2, and then later (line 28) list names.  I just wondered if it made more sense to list the names before the versions.